python/imzml_reader.py

```diff
@@ -171,26 +171,21 @@ def to_zarr(self, path, dtype=None, compressor=None, chunks=None):
         }
         z_arr.attrs["domain"] = self.domain
         z_arr.attrs["transform"] = self.transform
-        z_arr.attrs["dimensions"] = self.get_raster_dimensions()
+        z_arr.attrs["dimensions"] = self.get_image_dimensions()
 
 
 def write_raster_json(json_file, url, name, transform, dimensions):
-    raster_json = {
-        "schema_version": "0.0.1",
-        "images": [
-            {
-                "name": name,
-                "url": url,
-                "type": "zarr",
-                "metadata": {
-                    "dimensions": dimensions,
-                    "is_pyramid": False,
-                    "transform": transform,
-                },
-            }
-        ],
+    image_json = {
+        "name": name,
```

Comment:
`image.json` isn't consumed by vitessce, but rather nested within `raster.json` with the schema. I figured we don't need to version the intermediate images since the output is just the `raster`.